services/QuillLMS/app/controllers/api/v1/activity_questions_controller.rb

```diff
@@ -14,16 +14,16 @@ def index
   private def cached_questions(activity)
     Rails.cache.fetch(cache_key(activity), expires_in: 1.hour) do
       activity.questions.each_with_object({}) do |question, hash|
-        hash[question.uid] = question.as_json(locale: permitted_params[:locale])
+        hash[question.uid] = question.as_json(language: permitted_params[:language])
       end
     end
   end
 
   private def cache_key(activity)
-    "activity_questions/#{activity.uid}/#{permitted_params[:locale]}"
+    "activity_questions/#{activity.uid}/#{permitted_params[:language]}"
   end
 
   private def permitted_params
-    params.permit(:activity_id, :locale)
+    params.permit(:activity_id, :language)
```

Comment:
This is purely stylistic, so feel free to take or leave this, but I think this would be a little more in line with some of our newer style approaches if we extracted `permitted_params[:language]` into a private endless method:
```ruby
hash[question_uid] = question.as_json(language:)
...
"activity_questions/#{activity.uid}/#{language}"
...
private def language = permitted_params[:language]
```